docs/terraform.md

```diff
@@ -35,7 +35,7 @@
 | log_publishing_index_enabled | Specifies whether log publishing option for INDEX_SLOW_LOGS is enabled or not | bool | `false` | no |
 | log_publishing_search_cloudwatch_log_group_arn | ARN of the CloudWatch log group to which log for SEARCH_SLOW_LOGS needs to be published | string | `` | no |
 | log_publishing_search_enabled | Specifies whether log publishing option for SEARCH_SLOW_LOGS is enabled or not | bool | `false` | no |
-| name | Solution name, e.g. 'app' or 'jenkins' | string | `` | no |
+| name | Solution name, e.g. 'app' or 'jenkins' | string | `` | yes |
```

Comment:
```suggestion
| name | Solution name, e.g. 'app' or 'jenkins' | string | `` | no |
```
this is no how we should make variable mandatory. any way, let's keep it as is, but apply `aws_iam_role` logic fix